src/components/flame-graph/FlameGraph.js

```diff
@@ -90,17 +97,65 @@ class FlameGraph extends React.PureComponent<Props> {
     }
   };
 
+  /**
+   * Is the box for this call node wide enough to be selected?
+   */
+  _wideEnough = callNodeIndex => {
+    const { flameGraphTiming, callNodeInfo: { callNodeTable } } = this.props;
+
+    const depth = callNodeTable.depth[callNodeIndex];
+    const row = flameGraphTiming[depth];
+    const rowIndex = row.callNode.indexOf(callNodeIndex);
+    return row.end[rowIndex] - row.start[rowIndex] > SELECTABLE_THRESHOLD;
+  };
+
+  /**
+   * Return next keyboard selectable callNodeIndex along one
+   * horizontal direction.
+   *
+   * `direction` should be either -1 (left) or 1 (right).
+   *
+   * Returns undefined if no selectable callNodeIndex can be found.
+   * This means we're already at the end, or the boxes of all
+   * candidate call nodes are too narrow to be selected.
+   */
+  _nextSelectableInRow = (startingCallNodeIndex, direction) => {
```

Comment:
nit: please type the arguments and the result; especially you can type `direction` like `1 | -1`